oap-server/server-starter/src/main/resources/ui-initialized-templates/mysql/MYSQL-Root.json

```diff
@@ -0,0 +1 @@
+[{"id":"b6e3763c-882d-417f-8556-3cf0187cb8db","configuration":{"children":[{"x":0,"y":2,"w":24,"h":43,"i":"0","type":"Widget","graph":{"type":"ServiceList","dashboardName":"MYSQL-Service","fontSize":12,"showXAxis":false,"showYAxis":false,"showGroup":true}},{"x":0,"y":0,"w":24,"h":2,"i":"1","type":"Text","graph":{"fontColor":"blue","backgroundColor":"white","content":"Provide MYSQL monitoring.","fontSize":14,"textAlign":"left","url":"https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-mysql-monitoring/"}}],"layer":"MYSQL","entity":"All","name":"MYSQL-Root","isRoot":true}}]
```

Comment:
JSON should be well formated. You could use IDEA or an online tool to format. 
Then you should put the Apache license header on the top.

Check other dashboard configurations to follow up.

Reply:
I get it.